src/epochs.lua

```diff
@@ -61,7 +61,7 @@ local epochs = {}
 Epochs = Epochs or {}
 EpochSettings = EpochSettings
 	or {
-		pruneEpochsCount = 14, -- prune epochs older than 14 days
+		maxCachedEpochsCount = 14, -- prune epochs older than 14 days
```

Comment:
note: this will require manually setting this value in existing processes and removing the existing settings, given our use of `or`

Reply:
60bcdc8